src/main/java/org/broadinstitute/consent/http/service/DarCollectionService.java

```diff
@@ -144,4 +150,12 @@ public DarCollection updateDarCollection(Integer collectionId, User user) {
     return collections;
   }
 
+  private PaginationResponse<DarCollection> createEmptyPaginationResponse(Integer unfilteredCount) {
+    return new PaginationResponse<DarCollection>()
+          .setUnfilteredCount(unfilteredCount)
+          .setFilteredCount(0)
+          .setFilteredPageCount(1) //should this be 0 or 1?
```

Comment:
I'm conflicted on this point. `1` seems safer from a UI perspective, but `0` is more accurate. Let's stick with what you have and we can adjust if it causes problems.